packages/rest-typings/src/v1/Ajv.ts

```diff
@@ -4,6 +4,13 @@ import addFormats from 'ajv-formats';
 const ajv = new Ajv({
 	coerceTypes: true,
 });
+
 addFormats(ajv);
 
+ajv.addFormat('basic_email', /^[^@]+@[^@]+$/);
+ajv.addFormat(
+	'rfc_email',
+	/^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/,
```

Comment:
Where did this validation comes from?
I thought the correct RFC email validation regex was this one: https://pdw.ex-parrot.com/Mail-RFC822-Address.html

but I am not sure if we need to go this far though

Reply:
I believe you have already found the email validator as you have resolved this conversation, but I will comment here for future reference. These regexes were extracted from the emailValidator.ts in the apps/meteor/lib folder, which was implemented in this PR: https://github.com/RocketChat/Rocket.Chat/pull/23816. Since these were approved by Tasso and Diego, I believe they accurately represent RC's view on email validation.

I am aware that more robust regular expressions exist, and I encountered several during my research on the impacts of this change. However, since we already have an in-house solution that is used throughout most of the system, I decided to maintain it for consistency's sake.

Reply:
yea, I was looking into to see if we were being consistent,
once I found the email validator using the same regex to validate it I decided to delete, since I don't think we need to go over the top with e-mail validation since the actual only way for us to be 100% an e-mail is valid is to actually send stuff to it, also, the regex that is being used is good enough for all the cases I could've think about.

and since it is consistent there is no need to worry about this, good work :)